### PR TITLE
Module editable

### PR DESCRIPTION
**Pull request by DavidGeismarLtd**
This is a playground for trying progressively to move to a modular implementation of the CMS module (in fine a gem) : 
What I did
- All module related content is under Editable namespace
- created a rails generator `rails generate editables` to : 
a) create migration file
b) create the config file for the fields
c) create the models
d) add the route

let me know what you think we should do next :)

### CODE REVIEW

File: db/migrate/20200815082713_editable_fields.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class EditableFields < ActiveRecord::Migration[6.0]
+class Fields < ActiveRecord::Migration[6.0]
```
Comment (ManyBaby) [inline db/migrate/20200815082713_editable_fields.rb:1]: La migration risque de ne plus passer là non ?

**Comment by ManyBaby** [comment]
@davidgeismar la PR est trop grosse pour moi je vais tirer la branche est tester en local mais je n'ai pas le cerveau assez gros pour relire sereinement.

**Comment by davidgeismar** [comment]
on peut se faire un call pour que je te briefe en direct sur les differentes modifs

**Comment by DavidGeismarLtd** [comment]
> This is a playground for trying progressively to move to a modular implementation of the CMS module (in fine a gem) :
> What I did
> 
> * All module related content is under Editable namespace
> * created a rails generator `rails generate editables` to :
>   a) create migration file
>   b) create the config file for the fields
>   c) create the models
>   d) add the route
> 
> let me know what you think we should do next :)

**EDIT 20/09/2020**

---> working version that relies on [editables rails engine ](https://github.com/DavidGeismarLtd/content_editable).
All ruby files have been migrated 
Still need to migrate test files
Still need to migrate javascript files or create npm package ?
@ManyBaby if you want we can have a look at that when you are in Paris and merge it if you are satisfied too with the way this works

**Pull request merged**